cubedash/_model.py

```diff
@@ -57,70 +61,120 @@
         # release="myapp@1.0.0",
     )
 
+cache = Cache()
```

Comment:
In the future we should try and wrap up the initialisation of the sentry_sdk and SummaryStore as [flask extensions](https://flask.palletsprojects.com/en/3.0.x/extensiondev/), so that they can get configuration from the app, instead of loading files themselves or via environment variables.